modules/bmc/bmc_plugin.rb

```diff
@@ -1,7 +1,6 @@
 module Proxy::BMC
   class Plugin < Proxy::Plugin
-    http_rackup_path File.expand_path("http_config.ru", File.expand_path(__dir__))
-    https_rackup_path File.expand_path("http_config.ru", File.expand_path(__dir__))
+    rackup_path File.expand_path("http_config.ru", File.expand_path(__dir__))
```

Comment:
Reading this, I now wonder if we should also simplify this to:
```suggestion
    rackup_path File.expand_path("http_config.ru", __dir__)
```

That's equivalent, right?

Reply:
Seems like from testing that it should resolve to the same value.